### ISSUE TITLE
cdk-assets: log asset size before publishing

### ISSUE THREAD

**Issue by moltar** [aws/aws-cdk#18066]
### Description

Would be great to see the asset size in the log files.

https://github.com/aws/aws-cdk/blob/fd8e0e33816cb46678f7d1beac80b1623cdb6bac/packages/cdk-assets/lib/publishing.ts#L82-L84

### Use Case

Sometimes I want to see asset size increases / decreases and checked the logs assuming it would be reported. But it was not.

### Proposed Solution

Add both total bytes and human-readable sizes to the log.

https://github.com/aws/aws-cdk/blob/fd8e0e33816cb46678f7d1beac80b1623cdb6bac/packages/cdk-assets/lib/publishing.ts#L82-L84

### Acknowledge

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

**Comment by eladb** [comment]
Yes, could be a useful thing to add to `cdk-assets`. Contributions are welcome!
